infra/build/functions/project_sync_test.py

```diff
@@ -30,7 +30,7 @@
 from project_sync import get_projects
 from project_sync import ProjectMetadata
 from project_sync import sync_projects
-import test_utils
+from build.functions import test_utils
```

Comment:
Should I make the other imports non-relative?
I'm a bit scared to do it to the non-test modules in build/ since i don't know if the relative import is needed in the environment they are run in, but I can do it for other imports in test files.

Reply:
This is no longer relevant right?

In any case I think we ran into issues with relative imports with the non-test modules, because they had to be run outside of a package context.

Reply:
Yeah not relevant. I couldn't fix it :-(